update contribution section (#173)
zuplo/zudoku: README.md

## ⚙️ Installation

You can use the CLI to generate a new project or use it standalone via CDN as a React component.

### ⚡️ Quick start

> Fire up your new API docs using the command line generator:

Reviewer: ```suggestion
Fire up your new API docs using the create command:
```

```

> npm create-zudoku-app@latest

Reviewer: ```suggestion
npm create zudoku-app@latest
```

This is the correct way to do it. For your change to work it'd need to be `npx create-zudoku-app@latest`. It's confusing

## 🎓 Examples

- [Rick & Morty API](https://zudoku.dev/demo?api-url=https://rickandmorty.zuplo.io/openapi.json)
- [Pet Store API](https://zudoku.dev/demo?api-url=https://zudoku.dev/petstore.oas.json)
- [Zuplo API Documentation](https://docs-zudoku.pages.dev/)

### Zudoku use cases

> Because Zudoku is a flexible and highly customizable framework it can be used to create many things, including:

Reviewer: ```suggestion
Zudoku is a flexible and highly customizable framework that can be used to create many things, including:
```
